src/test/java/org/llorllale/cactoos/matchers/MatcherOfTest.java

```diff
@@ -55,4 +66,31 @@ public void mismatchesFunc() {
             new IsNot<>(new Matches<>(1))
         ).affirm();
     }
+
+    @Test
+    public void describesMismatch() {
+        this.exception.expect(AssertionError.class);
+        this.exception.expectMessage(
+            new UncheckedText(
+                new FormattedText(
+                    // @checkstyle LineLength (1 line)
+                    "describes mismatch%nExpected: \"Must be > 5\"%n but was: <1>"
+                )
+            ).asString()
+        );
+        new Assertion<>(
+            "describes mismatch",
+            1,
+            new MatcherOf<>(x -> x > 5, new TextOf("Must be > 5"))
+        ).affirm();
+    }
+
+    @Test
+    public void matchesAnyProc() {
+        new Assertion<>(
+            "wraps HasLines matcher",
```

Comment:
@vzurauskas what does HasLines have to do with this test?

Reply:
@llorllale Oh right, this is a leftover of something I was trying earlier. I rewrote the test like this now:

```java
@Test
public void matcherOfProcMatchesAnyArguments() {
    new Assertion<>(
        "matches any arguments when constructed from a Proc",
        new MatcherOf<>(String::trim),
        new Matches<>("a")
    ).affirm();
}
```